src/modules/vmount/output.cpp

```diff
@@ -214,8 +214,6 @@ void OutputBase::_calculate_angle_output(const hrt_abstime &t)
 		}
 	}
 
-	float dt = math::constrain((t - _last_update) * 1.e-6f, 0.001f, 1.f);
```

Comment:
intentional?

Reply:
Yes, because the RC input now defines either an angle, or an angular rate. In both cases it doesn't matter when the last update was, it's just an instantaneous setpoint.

Does that make sense?

Reply:
But doesn't the output value still need to be a proper angle?

Reply:
The output value sent in the command depends on the frame set in MOUNT_CONFIGURE, see:
https://github.com/PX4/PX4-Autopilot/blob/43a04c9bca31f31328c56a439931b7d6438897cb/src/modules/vmount/common.h#L71-L73
This means that this is in theory correct but there might of course be gimbals that might not listen to that.
All in all gimbal v1 is pretty broken, and that's why I was tasked with working on gimbal v2 :shrug:, so honestly I'm not sure what I should do here.

I guess I could try to revert this change and the hack setting the frame and see if it works.